main.css

```diff
@@ -0,0 +1,604 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;700&display=swap');
+
+html {
+  background: #191765;
+  display: flex;
+}
```

Comment:
- [ ] It would be so much better if the portfolio on the mobile version doesn't have a scroll bar at the bottom. This will make the app looks bad and the functionality is difficult. Please fix this. 

![Untitled](https://user-images.githubusercontent.com/64173189/133839147-79e31353-565b-4108-9305-f6b6fdfe3d23.png)


💪 😄